src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -656,4 +704,8 @@ public LDAPExtendedTemplate getLdapTemplate() {
         return ldapTemplate;
     }
 
+    private static boolean isPasswordCompliant(String password){
```

Comment:
This method is either badly named or is inverted from what it is named as. It returns `false` if the password is compliant?

Reply:
```suggestion
    private static boolean isPasswordNonCompliant(String password){
```